srs_data_management/generate_baseline_report.py

```diff
@@ -25,22 +28,30 @@ def checkout_model_repos(self, release):
         gmleng = Repo(constants.ASAPP_MLENG_ROOT)
         gmleng.git.checkout(release)
 
-        gcomcast = Repo(constants.ASAPP_COMCAST_SRS_ROOT)
-        gcomcast.git.checkout(release)
+        gclient = None
```

Comment:
Could we remove this? I like this pattern for `__init__` methods — it makes it clear that the class will define this variable below. In this instance, I don't see the same benefit.